include/lsst/afw/geom/Angle.h

```diff
@@ -407,9 +406,8 @@ inline Angle Angle::wrapNear(Angle const& refAng) const noexcept {
     if (wrapped - refAngRad >= PI) {
         wrapped -= TWOPI;
     }
-    // maximum relative roundoff error for subtraction is 2 epsilon
     if (wrapped - refAngRad < -PI) {
-        wrapped -= wrapped * 2.0 * std::numeric_limits<double>::epsilon();
+        wrapped += TWOPI;
```

Comment:
While this is certainly cleaner, I do wonder why the original code didn't use `TWOPI` here when it did everywhere else.